client/settings/view.js

```diff
@@ -6,15 +6,15 @@ import React, { PropTypes } from 'react';
 /**
  * Internal dependencies
  */
-import WCCSettingsForm from './wcc-settings-form';
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/1a7ee9b4d3d4dbf29363272d85e8b4722fbc50f9